src/services/dashboards/widgets/budget-usage-summary/BudgetUsageSummaryWidget.vue

```diff
@@ -110,6 +139,14 @@ interface Data {
     }
 }
 
+interface ChartData {
+    budget_type: string;
+    budget_rate: number;
+    pieSettings?: {
+        fill: string
```

Comment:
nitpick: semicolon seems to be missing
```suggestion
        fill: string;
```